website/docs/r/notification_channel.html.markdown

```diff
@@ -77,7 +77,7 @@ Each notification channel type supports a specific set of arguments for the `pro
   * `email` - (Required) Specifies the user email for integrating with Pagerduty.
   * `customDetails` - (Optional) Free text that *replaces* the content of the alert.
 * `SLACK`
-  * `channelId` - (Required) Specifies the Slack channel id. This can be found in slack browser via the url. Example - https://app.slack.com/client/<UserId>/<ChannelId>.
+  * `channelId` - (Required) Specifies the Slack channel id. This can be found in slack browser via the url. Example - https://app.slack.com/client/\<UserId>/\<ChannelId>.
```

Comment:
For the reviewer: Kindly refer to the description in the PR explaining why these changes were made.